docs/intro.md

```diff
@@ -36,7 +36,8 @@ The `type Query { }` block declares the type, `hello` is the field definition, `
 
 ## Validating schema
 
-Ariadne provides tiny `gql` utility function that takes single argument: GraphQL string, validates it and raises descriptive `GraphQLSyntaxError`, or returns the original unmodified string if its correct:
+Ariadne provides the `gql` utility function to validate schema. It that takes a
+single argument: a GraphQL string, like the following example.
```

Comment:
Is the linebreak in the middle of sentence purposeful here?

Reply:
It's markdown which tends to follow a convention of 80 characters per written line, for legibility in your favorite editor. When it renders in a page/browser, however, it looks continuous. Rendered line breaks require a blank line in the original content.

Reply:
I understand that, but we are not following this convention on Ariadne docs. Its much easier to edit when its single long paragraph wrapped by editor with single `>` sign in front of it.